### DISCUSSION TITLE
음소거 표시 위치 변경 건의

### DISCUSSION THREAD

**Discussion by neopandora** [metamong-friends/metamong-friends.github.io#42] [제안하기]
음소거 표시가 얼굴 표시되는 부분 아래에 있는데 표시되는 사람의 입부분이나 많은 부분을 가려서 잘 안보이는 것 같습니다. 머리 위 공간이나 다르게 표시되면 얼굴이 덜 가려질 것 같습니다.

**Comment by ACT-TJ** [comment]
안녕하세요!

유저가 상대의 마이크 음소거를 확인하는 빈도가 미팅을 진행하다보면 많다보니, 해당 아이콘 또한 잘 노출되는 위치를 고려하여 
자리하게 되었습니다. 해당 사유도 감안을 해주시면 좋을 것 같고 익숙해지시면 얼굴 위치를 좀 올려서 쓰시지 않을까 싶어요^^.
저희도 좀 더 나은 안이 있는지 검토해 보도록 하겠습니다^^
